modules/Hoot/ui/conflicts/merge.js

```diff
@@ -38,28 +52,20 @@ export default class Merge {
      * @returns {Promise<void>}
      */
     async mergeFeatures() {
-        let features = _clone( this.data.currentFeatures ),
-            reverse  = d3.event.ctrlKey || d3.event.metaKey || this.mergeArrow.reverseFeatures,
-            featureUpdate,
-            featureDelete,
+        let reverse  = d3.event.ctrlKey || d3.event.metaKey && !this.isPoiPoly,
+            featureUpdate = _cloneDeep((reverse) ? this.mergeArrow.from : this.mergeArrow.to),
+            featureDelete = _cloneDeep((reverse) ? this.mergeArrow.to : this.mergeArrow.from),
```

Comment:
@jackgrossman18 we have to deep clone here because we modify the feature tags on line 68.  If we do a merge, then undo it and reverse it the element merge service errors because both featuers have the `hoot:merge:target=yes` tag.

Reply:
Also when checking if reverse merge is valid, we disallow if it is POI->Poly.  We can simply disallow because code in `checkMergeButton` ;omr 358 makes sure the poi and poly features are always in the right order.